applications/kserve/0.15.0/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ./defaults
```

Comment:
(nit) 
there is really no need for this defaults dir here. can we move the cm.yaml to live next to the helmrelease it is used in? 

this looks fine as-is if you have a good reason to do that.